src/commonTest/kotlin/fr/acinq/lightning/channel/states/WaitForChannelReadyTestsCommon.kt

```diff
@@ -71,12 +71,12 @@ class WaitForChannelReadyTestsCommon : LightningTestSuite() {
         val (alice, channelReadyAlice, bob, channelReadyBob) = init()
         val (alice1, actionsAlice1) = alice.process(ChannelCommand.MessageReceived(channelReadyBob))
         assertIs<LNChannel<Normal>>(alice1)
-        assertEquals(2, actionsAlice1.size)
+        assertEquals(3, actionsAlice1.size)
```

Comment:
It would be useful to explicitly test that we're sending the `Confirmed` event here - to be added in a next PR whenever we have the occasion

Reply:
Good catch, noted